modules/services/espanso.nix

```
@@ -46,6 +63,27 @@ in
         defaultText = literalExpression "pkgs.espanso";
       };
 
+      package-wayland = mkOption {
+        type = types.package;
+        description = "Which espanso package to use when running under wayland on linux";
+        default = pkgs.espanso-wayland;
+        defaultText = literalExpression "pkgs.espanso-wayland";
+      };
```

```suggestion
      package-wayland = mkPackageOption pkgs "espanso-wayland {
        nullable = true;
        extraDescription = "Which `espanso` package to use when running under wayland.";
      } // { 
        default = if isLinux && cfg.waylandSupport then pkgs.espanso-wayland else null; };
```

We should utilize `mkPackageOption` and perhaps just null it out on darwin, since it never applies.